src/sync_drive.py

```diff
@@ -90,11 +90,12 @@ def package_exists(item, local_package_path):
             )
             return True
         else:
-            LOGGER.debug(
+            LOGGER.info(
                 f"Changes detected: local_modified_time is {local_package_modified_time}, "
                 + f"remote_modified_time is {remote_package_modified_time}, "
                 + f"local_package_size is {local_package_size} and remote_package_size is {remote_package_size}."
             )
+            rmtree(local_package_path)
```

Comment:
Without removing already downloaded/extracted package we couldn't download new copy of it because file already exists and `unpack_archive` raised exception `OS error 21`.

We will download whole package in few moments anyway, so just delete old version.

Reply:
Makes sense.